morpho-checkpoint-node/src/Morpho/Tracing/Tracers.hs

```diff
@@ -73,15 +71,6 @@ data Tracers peer localPeer h c = Tracers
     powNodeRpcTracer :: forall i o ev. (Show ev, ToJSON ev, HasSeverityAnnotation ev) => Tracer IO (RpcTrace ev i o),
     extractStateTracer :: Tracer IO (ExtractStateTrace h c),
     timeTravelErrorTracer :: Tracer IO (TimeTravelError (MorphoBlock h c)),
-    -- | Chain Tip tracer.
-    --
-    --   Note: we're currently not using structured logging approach
-    --   for the chain Tip tracer. Implementing all the required
-    --   typeclasses is a hassle and we're not mining these
-    --   structured logs anyways, we are just plain-text quering
-    --   them. Modeling the chain tip hash as Text should be good
-    --   enough for now.
-    chainTipTracer :: Tracer IO Text,
```

Comment:
What happened to the chainTipTracer?

Reply:
See https://github.com/input-output-hk/ECIP-Checkpointing/pull/34/commits/eda2549354774d9c61e9846bccf18825e7a7dac3

Removed because it's already covered with an ouroboros tracer